linetools/tests/test_xspectrum1d.py

```diff
@@ -0,0 +1,20 @@
+# Module to run test on XSpectrum1D
+
+from __future__ import print_function,absolute_import,division,unicode_literals
+import numpy as np
+import glob,os,sys,copy
+from astropy.table import QTable,table
+from astropy.io import ascii
+from astropy import units as u
+from astropy import constants as const
+from matplotlib import pyplot as plt
+from enigma.qpq import spec as qpqs
```

Comment:
linetools can't depend on enigma

And Travis usually can't handle plotting tests